### PR TITLE
feat(TDP-1716): remove react native from utils

### PR DESCRIPTION
**Pull request by adamosborne-tnl**
Removed View, Dimension & PixelRation react-native from utils [TDP-1716](https://nidigitalsolutions.jira.com/browse/TDP-1716)

### CODE REVIEW

File: packages/pagination/__tests__/web/pagination-with-styles.web.test.js [hunk 1]
```diff
@@ -8,24 +7,11 @@ import {
...
-  "height",
-  "flexDirection",
-  "fontFamily",
-  "fontSize",
-  "marginRight",
-  "paddingTop",
-  "paddingVertical",
-  "textAlign"
-];
+import "jest-styled-components";
```
Comment (adamosborne-tnl) [inline packages/pagination/__tests__/web/pagination-with-styles.web.test.js:14]: changed this from the request() as for some very strange reason it was complaining about a missing }

File: packages/article-main-standard/__tests__/web/__snapshots__/article-with-style.web.test.js.snap [hunk 2]
```diff
@@ -1,6 +1,24 @@
...
 
 exports[`full article with style 1`] = `
+.c26 {
+  border: 0px solid black;
+  box-sizing: border-box;
+  display: -webkit-box;
+  display: -webkit-flex;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-flex-direction: column;
```
Comment (DMackintosh-TNL) [inline packages/article-main-standard/__tests__/web/__snapshots__/article-with-style.web.test.js.snap:11]: Do we not use AutoPrefixer to create vendor-prefixes like this, as we shouldn't have to write these?

Comment (adamosborne-tnl) [reply packages/article-main-standard/__tests__/web/__snapshots__/article-with-style.web.test.js.snap:11]: Thankfully I didn't write them, those were just generated in the snapshot

File: packages/link/src/link.js [hunk 3]
```diff
@@ -13,7 +13,7 @@ const respStylesSelector = selector => ({ responsiveLinkStyles }) =>
   (responsiveLinkStyles && responsiveLinkStyles[selector]) || "";
 
 const RespLink = styled.a`
-  text-decoration: ${props =>
+  "text-decoration": ${props =>
```
Comment (flashcheeks) [inline packages/link/src/link.js:16]: Are the `"` needed?

**Review by flashcheeks — APPROVED** [review]

File: packages/link/src/link.js (hunk 3 above)
Comment (adamosborne-tnl) [reply packages/link/src/link.js:16]: yes, otherwise it failed linting

**Review by yavorpeev1 — APPROVED** [review]

**Review by DMackintosh-TNL — APPROVED** [review]

**Review by SunshineEleda — APPROVED** [review]

**Pull request merged**